### ISSUE TITLE
core_components.story.exs broken

### ISSUE THREAD

**Issue by LostKobrakai** [phenixdigital/phoenix_storybook#251]
It still uses the rc field input syntax of `field={{f, …}}`.

The story rendered blank without showing an error and when trying to look at the source it showed an phoenix error. I think the rendering of the story should at least show that there was an error.

**Comment by LostKobrakai** [comment]
Where I saw the issue has been "0.5.0"
```
"phoenix_storybook": {:hex, :phoenix_storybook, "0.5.0", "c70ae33f53adf54e0cb99271e7617cd5160d1b6780fc7bc50b6da2893bfbde3c", …}
```

**Comment by cblavier** [comment]
I could reproduce the issue and just fixed it. 
It will be released in 0.5.2
